### DISCUSSION TITLE
Props in stitches component

### DISCUSSION THREAD

**Discussion by ovinidev** [stitchesjs/stitches#957] [Help]
How to make this with Stitches?

![image](https://user-images.githubusercontent.com/62653539/154264295-c46ae78f-1592-49be-b444-440018dcff09.png)

![image](https://user-images.githubusercontent.com/62653539/154264333-320e7e7e-a81f-43da-b318-62821fdfe851.png)

![image](https://user-images.githubusercontent.com/62653539/154264424-56344cd7-cbff-4a02-ae01-181c4cb97a62.png)

I'm passing a new property to my component, I want to pass an image url in the component into the styles file so I can use this url in the backgroundImage.

**Comment by hadihallak** [comment]
Dynamic props aren't allowed in stitches. if you need something dynamic, use the [css prop](https://stitches.dev/docs/overriding-styles) or use react inline styles

**Reply by ovinidev** [reply]
Thanks man, you think about to implementing this in Stitches?

**Reply by hadihallak** [reply]
Don't think so. Not having dynamic props is one of the main things that the Stitches API was designed around.

**Reply by ovinidev** [reply]
True, I'm using Stitches in my projects and I'm loving it
